src/Datadog.AutoInstrumentation.NativeLoader/dynamic_dispatcher.cpp

```diff
@@ -0,0 +1,332 @@
+#include "dynamic_dispatcher.h"
+
+#include <filesystem>
+#include <fstream>
+#include <unordered_map>
+
+#include "logging.h"
+#include "pal.h"
+
+#if AMD64
+
+#if _WINDOWS
```

Comment:
Mentioned before, but conversation got los: Make this indented?

Reply:
why ? As i said in the previous comment, we did not agreed on style like this so leave this for later discussion